src/sentry/conf/server.py

```diff
@@ -1595,7 +1600,24 @@ def create_partitioned_queues(name):
         "pull": True,
         "ports": {"5432/tcp": 5432},
         "environment": {"POSTGRES_DB": "sentry", "POSTGRES_HOST_AUTH_METHOD": "trust"},
-        "volumes": {"postgres": {"bind": "/var/lib/postgresql/data"}},
+        "volumes": {
+            "postgres": {"bind": "/var/lib/postgresql/data"},
+            "wal2json": {"bind": "/wal2json"},
+            CDC_CONFIG_DIR: {"bind": "/cdc"},
+            os.path.join(CDC_CONFIG_DIR, "init_hba.sh"): {
```

Comment:
This is because, if the DB is not yet initialized we cannot do the initialization in the entrypoint, that is done through the initdb.d scripts. Which are instead not executed if the DB is already initialized.

Reply:
DB initialized --> /cdc/postgres-entrypoint.sh
DB *not* initialized --> /docker-entrypoint-initdb.d/init_hba.sh

Right?

Reply:
If the DB is *not* initialized, does this take `SENTRY_USE_CDC_DEV` into account?

Reply:
> Right?

Correct. Unfortunately the standard postgres image does not allow a unified way to initialize the server in both cases.

> If the DB is not initialized, does this take SENTRY_USE_CDC_DEV into account?

I don't think this is possible. But I am not too concerned if we setup pg-hba even without cdc running.

Reply:
Actually there should be a way to do this by not mounting the file at all if SENTRY_USE_CDC_DEV is not set. Will change.

Reply:
done